### PR TITLE
Improve tutorial documentation

### PR DESCRIPTION
**Pull request by Codym48**
Fix documentation inconsistencies encountered when executing the https://nasa.github.io/trick/tutorial/Tutorial

Add newline after error message to create more readable output like

    cmartin@docker-desktop:~/trick-tutorial$ trick-CP
    S_define does not exist
    cmartin@docker-desktop:~/trick-tutorial$

instead of

    cmartin@docker-desktop:~/trick-tutorial$ trick-CP
    S_define does not existcmartin@docker-desktop:~/trick-tutorial$

### CODE REVIEW

**Comment by Codym48** [comment]
Poking @jdeans289 and @spfennell as recent contributors ... let me know if there's anything else I can do!

File: docs/tutorial/ATutIntroduction.md [hunk 1]
```diff
@@ -68,27 +68,5 @@ its capabilities.
 If Trick is not already installed on your machine, then you will need to do that
 first, by following the directions at: [Install Guide](/trick/documentation/install_guide/Install-Guide).
 
-Once Trick is installed on your machine, you will need add the Trick **bin**
```
Comment (jdeans289) [inline docs/tutorial/ATutIntroduction.md:71]: Why did you remove this section?

**Review by jdeans289 — COMMENTED** [review]
Most of this looks good to me! My only concern is removing the section on adding the Trick bin directory to path. @jmpenn will have final say on what goes into the tutorial, but most of these changes seem necessary to me.

File: docs/tutorial/ATutIntroduction.md (hunk 1 above)
Comment (Codym48) [reply docs/tutorial/ATutIntroduction.md:71]: This page already links to [the installation guide](https://github.com/Codym48/trick/blob/cleanup-docs/docs/documentation/install_guide/Install-Guide.md#4-optionally-update-your-environment), which seems to contain the latest and greatest information regarding environment variables and specifically advises against setting `TRICK_HOME` and friends,

I could restore a sentence about optionally modifying `PATH`?

Comment (Codym48) [reply docs/tutorial/ATutIntroduction.md:71]: Ok, I put the content back but made it obvious that PATH modification is optional rather than required and left TRICK_HOME out.

Comment (jdeans289) [reply docs/tutorial/ATutIntroduction.md:71]: Thanks! We'll get this merged now

**Review by jdeans289 — APPROVED** [review]

**Comment by jdeans289** [comment]
Thank you for your contribution!

**Comment by Codym48** [comment]
> Thank you for your contribution!

Happy to help!
